package.json

```diff
@@ -34,10 +32,15 @@
     "eslint-plugin-jest": "^24.4.0",
     "eslint-plugin-prettier": "^3.4.0",
     "jest": "^27.0.1",
+    "lerna": "^4.0.0",
+    "prettier": "^2.3.2",
     "ts-jest": "^27.0.4",
     "ts-node": "^10.1.0",
     "ts-node-dev": "^1.1.8",
     "tsconfig-paths": "^3.10.1",
     "typescript": "^4.5.5"
+  },
```

Comment:
Please move this import to the `ui` package.json!

I'm assuming you accidentally npm install'd in root.
- also please remove the package-lock.json (that's a result of npm install but can cause conflicts with our yarn.lock :))

Reply:
It's no longer used so I deleted it.